src/universal/modules/newTeam/components/NewTeamForm/NewTeamForm.js

```diff
@@ -93,8 +92,9 @@ class NewTeamForm extends Component {
 
     return (
       <form className={css(styles.form)} onSubmit={handleSubmit(this.onSubmit)}>
-        <Panel label="Create a New Team">
+        <Panel hasHeader={false}>
```

Comment:
nit: consider making `hasHeader` default to false. it's always nice when attributes add things instead of take them away.

Reply:
got it! thanks